app/src/NativeBase/Theme/StaTheme.tsx

```diff
@@ -28,7 +28,8 @@ const common = {
   },
   fonts: {
     family: {
-      primary: 'Poppins-Regular',
+      primary: 'Poppins',
```

Comment:
I'm seeing an error message in my terminal: `fontFamily "Poppins" is not a system font and has not been loaded through Font.loadAsync.`

Reply:
🧐

Reply:
In the new setup with Expo, fonts get defined in [app/App.tsx](https://github.com/Scottish-Tech-Army/Volunteer-app/blob/main/app/App.tsx) and it's named `'Poppins-Regular'` there so I'm pretty sure that's what we should be using -- but was there a reason you changed it?

Reply:
The reason for changing this is that the `fontWeight` prop had no effect on the actual appearance of the fonts. This is resolved by replacing `Poppins-Regular` which is 400 weight, with `Poppins`. I tried this as it matches the suggested font set up in the Native Base docs. 
However, I do notice the error message now too and it's not there when the app is run from `main` so it's obviously not liking it. Not sure how to fix. Will do some digging later today.

Reply:
That's it fixed now. The error resolved by sticking to `Poppins-Regular` and adding semantic fonts to enable semiBold Bold and Light variations.

Reply:
Nice one -- that's a good solution 👍